kubernetes-data-platform/values.yaml

```diff
@@ -1,5 +1,5 @@
 presto:
-  workers: 2
+  workers: 0
```

Comment:
ambivalent here; technically presto can use the coordinator as a worker for something like local development but i believe there's a flag you have to set in the presto config to say "assign workloads to the coordinator" if you're going that route; alternatively i could make the argument of having a single worker by default to more closely mirror a "prod-like" scenario.

Reply:
We set it up this way to minimize the amount of system resources the cluster consumes. It's already heavy as it is.

Reply:
that's why i was on the fence; don't feel strongly either way.